benchpress/templates/nginx.conf

```diff
@@ -0,0 +1,117 @@
+
+#user  nobody;
+worker_processes  1;
```

Comment:
The default here might need to be changed as well. Look up in the install script?

Reply:
I was looking at the benchmark scripts you sent me a while back, I couldn't find conf settings for nginx itself, except for a comment to change the port other than port 80.

But, if you have suggestion let me know.